CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v0.42.0 [2021-12-08]
+
+_What's new?_
+
+- New tables added
+  - [aws_ec2_managed_prefix_list](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_ec2_managed_prefix_list) ([#813](https://github.com/turbot/steampipe-plugin-aws/pull/813))
+  - [aws_vpc_peering_connection](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_vpc_peering_connection) ([#814](https://github.com/turbot/steampipe-plugin-aws/pull/814))
+
+_Enhancements_
+
+- Added column `prefix_list_id` to `aws_vpc_security_group_rule` table ([#801](https://github.com/turbot/steampipe-plugin-aws/pull/801))
+- Added column `compliance_by_config_rule` to `aws_config_rule` table ([#817](https://github.com/turbot/steampipe-plugin-aws/pull/817))
+- Added additional optional key quals, filter support, and context cancellation handling to `Access Analyzer`, `ACM`, `API Gateway`, `Application Auto Scaling`, `Audit manager`, `Backup`, `Cloud Control`, `CloudFormation`, `CloudFront`, `CloudWatch`, `CodeBuild`, `CodeCommit`, `CodePipeline`, `Config`, `DAX`, `Directory Service`, `DMS`, `DynamoDB` and `EBS` tables   ([754](https://github.com/turbot/steampipe-plugin-aws/pull/754))
+- Added an example query in the `aws_rds_db_instance` table document ([#806](https://github.com/turbot/steampipe-plugin-aws/pull/806))
```

Comment:
```suggestion
- Added an example query for listing MySQL instances with SSL disabled in the `aws_rds_db_instance` table document ([#806](https://github.com/turbot/steampipe-plugin-aws/pull/806))
```

Reply:
Also, in the table doc, the desc. should read `### List MySQL instances with SSL disabled in assigned parameter group` (https://github.com/turbot/steampipe-plugin-aws/pull/806/files#diff-3d9e194008ec43e1eb616a90e1dae15d5b2f7f36df9a760e347bf8dab751ae9eR108), can you please update that before releasing?